agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -0,0 +1,50 @@
+#script to log and readout pfeiffer TPG 366 gauge contoller
+#via Ethernet connection
+#Zhilei X., Tanay B.
+
+import socket
+import numpy as np
+
+BUFF_SIZE = 128
+ENQ = '\x05'
+IP_ADDRESS = '10.10.10.20'
+
+class pfeiffer:
+    def __init__(self, ip_address=IP_ADDRESS, port=8000, timeout=10):
```

Comment:
Docstring missing. Please describe the class, its arguments, and attributes.

Reply:
Done

Reply:
Formatting is important here for Sphinx to format it correctly, again a good example can be found here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

The "Attributes" you have listed are the methods of the class, but what is meant by attributes is the variables you've stored for the class, for example, `self.ip_address`, `self.port`, and `self.comm`.